src/core/serializers/serializePlanInput.ts

```diff
@@ -112,6 +112,9 @@ const serializeProperties = (properties: Properties, chargeModel: ChargeModelEnu
             : undefined,
         }
       : { perTransactionMinAmount: undefined, perTransactionMaxAmount: undefined }),
+    ...(chargeModel === ChargeModelEnum.Custom
+      ? { customProperties: JSON.parse(properties?.customProperties) }
```

Comment:
There is an associated test for this file. Maybe it's worth checking this transform?

Reply:
@ansmonjol Oups my bad! See 833e8ef6333f53bb2e29d2d8077b608091050e83